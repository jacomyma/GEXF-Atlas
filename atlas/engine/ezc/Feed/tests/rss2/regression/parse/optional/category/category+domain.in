<?xml version="1.0" encoding="utf-8"?>
<rss version="2.0">
  <channel>
    <category domain="Category domain">Feed category</category>
  </channel>
</rss> 
