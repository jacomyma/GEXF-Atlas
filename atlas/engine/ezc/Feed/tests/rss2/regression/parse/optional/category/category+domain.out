<?php
$feed = new ezcFeed( 'rss2' );

$category = $feed->add( 'category' );
$category->term = 'Feed category';
$category->scheme = "Category domain";

return $feed;
?>
